ledger/participant-integration-api/src/main/scala/platform/store/ActiveLedgerStateManager.scala

```diff
@@ -159,9 +159,7 @@ private[platform] class ActiveLedgerStateManager[ALS <: ActiveLedgerState[ALS]](
                 witnesses = disclosure(nodeId),
                 // A contract starts its life without being divulged at all.
                 divulgences = Map.empty,
-                key = nc.versionedKey.map(
```

Comment:
I assume those checks don’t matter because we do them in the engine already?

Reply:
Yes the `assertNoCid` was used to change the type from `Value[ContractId]` to `Value[Nothing]`. Here I assume that the engine already check the absence of contract IDs.